### ISSUE TITLE
Landing page: Include version number in translation string

### ISSUE THREAD

**Issue by jennur** [inveniosoftware/invenio-app-rdm#1231]
Version number should be included in the translation string as a substitution variable to make sure the text get extracted correctly for all languages.

__Code__ (invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html)
`<span class="label text-muted"> | {{ _('Version') }} {{ record.ui.version }}</span>`

__Screenshot__
<img width="287" alt="Screenshot 2022-02-09 at 09 51 03" src="https://user-images.githubusercontent.com/21052053/153197777-2c842ad2-6acc-4d6c-a4ee-d8986f0e8ae0.png">
